Developer.md

```diff
@@ -0,0 +1,28 @@
+# Building and Testing the Java Client Core Library
+
+## Build/Dev Environment Prerequisites 
+ - [Java version 8+ e.g Temurin distribution](https://projects.eclipse.org/projects/adoptium.temurin/downloads)
+ - [Apache Maven version 3.8.5+](https://maven.apache.org/download.cgi)
+ - JUnit 5 Java Testing Framework version version 5.8+
+   - Note: Junit Testing Framework is a public java library that can be imported by adding the dependency to the pom.xml file
+ - [IntelliJ Community Edition Version 2022.1.3+ IDE](https://www.jetbrains.com/idea/download/#section=windows)
+
+
+## How to build this project
+ - Clone this repository, open the project in intelliJ IDE
```

Comment:
As mentioned, IntelliJ is not required. Building and running the project is just what a standard Maven project would be.

Reply:
included